src/scss/globals.scss

```diff
@@ -0,0 +1,375 @@
+// Start of Flex classes.
+.flex {
+  display: flex;
+}
+.flex-wrap {
+  flex-wrap: wrap;
+}
+.flex-grow {
+  flex-grow: 1;
+}
+// End of Flex classes.
+
+
+// Start of Global styles.
+.inline {
+  display: inline-block;
+}
+
+dl {
+  margin: 0;
+}
+
+// End of Global styles.
+
+
+// Start of header styles.
+.pw-calc {
+  margin: auto;
+  color: #333333;
+}
+.pw-calc-header__title {
+  font-weight: bold;
+  border-bottom: 5px solid #dce6eb;
+  padding-bottom: 1rem;
+}
+// End of header styles.
+
+
+// Start of  Input styles.
+.input {
+  width: 100%;
+  .form-item {
+    padding: 1rem 0;
+    .form-item__title {
+      padding-bottom: 0.5rem;
+    }
+  }
+}
+.form-item__title {
+  font-weight: bold;
+}
+.form-item {
```

Comment:
Just a style preference refactor if you have time.

1. Leave a line space between nested blocks.
```
.example {
  color: red;

  .next-one {
    color: blue;
  }
}
```

2. With scss and bem we can do something like
```
.block-name {
  color: red;

  &__element {
    color: blue;

    &--modifier
  }

  &__element-two {
    color: pink;
  }
}
```